blocks/suggest/suggest.bemhtml

```diff
@@ -21,6 +21,7 @@ block('suggest')(
                     target : 'anchor',
                     autoclosable : true
                 },
+                attrs: { 'aria-hidden': "true" },
```

Comment:
the bug is actually somewhere between `bem-xjst` and `bem-components` but I added it here to make spec pass until we fix it elsewhere